modules/efk-logging-fluentd-external.adoc

```diff
@@ -13,6 +13,9 @@ hosted Fluentd has processed them.
 ifdef::openshift-origin[]
 The `secure-forward` plug-in is provided with the Fluentd image as of v1.4.0.
 endif::openshift-origin[]
+ifdef::openshift-enterprise[]
+The `secure-forward` plug-in is supported by Fluentd only.
+enddef::openshift-enterprise[]
 
```

Comment:
@richm Is the `secure-forward` plug-in for Fluentd only?

Reply:
Yes.  We'll have to figure out some other way for rsyslog to forward logs off of the cluster.